src/Sulu/Bundle/MediaBundle/Api/Collection.php

```diff
@@ -40,19 +41,73 @@
  *          parameters = { "collection" = "expr(object.getId())", "limit" = 9999, "types" = "{types}" }
  *      )
  * )
+ * @Relation(
+ *      "self",
+ *      href = @Route(
+ *          "get_collection",
+ *          parameters = { "id" = "expr(object.getId())" }
+ *      )
+ * )
+ * @Relation(
+ *      "admin",
+ *      href ="expr('media/collections/edit:'~object.getId()~'/files')"
```

Comment:
We can't merge this PR really soon because another project doesn't work with these changes... Instead we will create a 0.17.0-RC1 from this branch, so maybe it is possible to change before the merge into develop.